exporter/googlecloudexporter/factory.go

```diff
@@ -38,10 +34,6 @@ const typeStr = "googlecloud"
 
 // NewFactory creates a factory for the googlecloud exporter
 func NewFactory() component.ExporterFactory {
-	// Hard code to use legacy config for now until we can fully test and understand new OTLPDirect behavior.
-	// Feature flag is in an init function so overridding must occur after inits are processed.
-	featuregate.GetRegistry().Apply(map[string]bool{"exporter.googlecloud.OTLPDirect": false})
```

Comment:
@jmwilliams89 does this mean the feature gate will be on or off for us?

Reply:
The default of OTLPDirect is true now. We had to add this before to force it to be false, which forced the exporter to use the legacy config.